ci/logging_tests_ctlplane.yml

```diff
@@ -0,0 +1,98 @@
+- name: Prepare Logging Tests
+  hosts: controller
+  gather_facts: false
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+
+- name: "RHOSO-12753"
+# description: Verify ctlplane node logging containers
+  hosts: computes
+  gather_facts: no
+  vars:
+    container_out_file: "verify_ctlplane_node_logging_containers_{{ inventory_hostname }}_lresults.log"
+    container_list:
+      - ceilometer_agent_compute
+      - ceilometer_agent_ipmi
+      - node_exporter
+
+  tasks:
```

Comment:
You  could combing these plays into one and define all the vars together.

Then you can delete the files all in one go (by deleting the output_dir if you are using it) and also copy the files to the hypervisor in one go.

As far as I know, zuul fetches files from each host at the end of a job, so if you define the right output_dir, then you don't have to worry about fetching the files at the end of the playbook.

Reply:
added main.yml that will trigger on unique lists for each task file defined.